tests/regression/src/lib.rs

```diff
@@ -86,7 +84,11 @@ mod tests {
         }
     }
 
-    fn valgrind_test<F>(test_name: &str, test_body: F) -> Result<(), s2n_tls::error::Error>
+    fn valgrind_test<F>(
```

Comment:
It might be better to make this a builder so when new config params get added in the future, it's easy to add them without breaking the harness. I'm biased but I think bolero's interface is nice for this:

https://github.com/camshaft/bolero/blob/b3ca02f4d563236ccc101622214ce64c5d3be32d/examples/rle-stack/src/tests.rs#L91

Reply:
I see the value in this, especially when we think about extending the usage of this tool beyond the current valgrind interface. However, given the limited scope and timeframe of this project, I think it would be better as a stretch goal. I can work on it towards the end of the internship or provide some insight in a Future Maintenance/Project Backlog document.

Reply:
That's fine